Userland/Services/Taskbar/ClockWidget.cpp

```diff
@@ -181,7 +212,7 @@ void ClockWidget::paint_event(GUI::PaintEvent& event)
     const int widget_width = max_width();
     const int translation_x = (widget_width - ideal_width) / 2 - frame_width;
 
-    painter.draw_text(frame_inner_rect().translated(translation_x, frame_width), time_text, font, Gfx::TextAlignment::CenterLeft, palette().window_text());
+    painter.draw_text(frame_inner_rect().translated(m_location == Gfx::Alignment::Bottom || m_location == Gfx::Alignment::Top ? translation_x : 0, frame_width), time_text, font, align, palette().window_text());
```

Comment:
If we're sure about having the separate `Alignment`, then the `TextAlignment` -> `Alignment` parts of this change should probably happen in the previous commit.

Reply:
This is part of the Taskbar vertical orientation bring-up so doesn't really make sense to change it in the previous comment.